### PR TITLE
tone equalizer interactive cursor : try to improve issue #3428

### PR DESCRIPTION
**Pull request by aurelienpierre**
Make the "fill" setting round to avoid confusion between the parameter intensity feedback, and the actual cursor:
Capture d’écran du 2019-11-14 15-44-09
(note that mouse arrow is added on the screenshot, it's not there in darktable).

### CODE REVIEW

**Comment by TurboGit** [comment]
Works for me. It was not an issue for me before either. Would be nice to have feedback from others.

**Comment by aurelienpierre** [comment]
Capture d’écran du 2019-11-15 13-59-57

**Comment by TurboGit** [comment]
Looks even better !

**Comment by aurelienpierre** [comment]
Capture d’écran du 2019-11-15 14-52-55

**Comment by Nilvus** [comment]
> Capture d’écran du 2019-11-15 14-52-55
> 
> People complained it looked like a penis

Let them complain ;-) ;-)

**Comment by aurelienpierre** [comment]
Well, once you have been told, you keep seeing it as a penis. So, I think the v3 will be less distracting.

**Comment by TurboGit** [comment]
> Capture d’écran du 2019-11-15 14-52-55
> 
> People complained it looked like a penis

Yes, let them complain, I prefer with the circle ! Or you may add a bar perpendicular to the arc ?

**Comment by Nilvus** [comment]
> Well, once you have been told, you keep seeing it as a penis. So, I think the v3 will be less distracting.

Just a joke ;-)

**Comment by aurelienpierre** [comment]
> Yes, let them complain, I prefer with the circle ! Or you may add a bar perpendicular to the arc ?

I would be careful with additional sharp details, since the cursor is hovering the picture and background/foreground interactions can really work for the best or for the worst. If the arc is not clear enough, I would rather make it thicker.

**Comment by TurboGit** [comment]
Ok, let's go with that then.

File: src/iop/toneequal.c [hunk 1]
```diff
@@ -2374,14 +2374,19 @@ void gui_post_expose(struct dt_iop_module_t *self, cairo_t *cr, int32_t width, i
   // set custom cursor dimensions
   const double outer_radius = 16.;
   const double inner_radius = outer_radius / 2.0;
-  const double setting_scale = 2. * outer_radius / zoom_scale;
+  //const double setting_scale = 2. * outer_radius / zoom_scale;
```
Comment (TurboGit) [inline src/iop/toneequal.c]: Please remove commented out code. Without a comment this will not help anyway.

Comment (aurelienpierre) [reply src/iop/toneequal.c]: oops, sorry.

**Comment by GLLM** [comment]
> Well, once you have been told, you keep seeing it as a penis. So, I think the v3 will be less distracting.

LOL ! I had not seen it that way ... but once told so, it's hard indeed to not see it again :-)

**Pull request merged**